pkg/plugin/test-keeper/plugin/test_status_service.go

```diff
@@ -39,7 +39,7 @@ const (
 )
 
 func (gh *GitHubTestEventsHandler) newTestStatusService(log log.Logger, change scm.RepositoryChange) testStatusService {
-	statusContext := github.StatusContext{BotName: "alien-ike", PluginName: ProwPluginName}
+	statusContext := github.StatusContext{BotName: gh.BotName, PluginName: ProwPluginName}
```

Comment:
So it was hardcoded to `alien-ike` and now it `plugin_init` we make it 

```
flag.String("bot-name", "ike-plugins", "Bot Name used for the plugin.")
```

Why?

Reply:
Shouldn't the `BotName` be consistent across the plugins? If we want to have hardcoded values then why make it configurable?

Reply:
Ok I might not be clear in the morning. Sorry, very little sleep this night.

What I meant is we should make it `alien-ike` across all the plugins so the default should be changed :) (have a look at the checks below, `BotName` is first part of the `xxx/yyy` naming we have for the plugins and since we are using @alien-ike account it should be configured to that. For the moment defaulting it is fine.

Reply:
I have discussed it with Hemani yesterday and this was kind of my idea - I understood the issue that all occurrences of the `BotName` should be same and equal to what is set in the flag - sorry for confusions :-)